pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
Let's don't commit `pnpm-lock.yaml` if you don't change any dependencies. You can use `corepack enable` to make sure that you have the same PNPM version as described in the [package.json](https://github.com/remirror/remirror/blob/30ff3ddc073125fac7f7a354802dcbea57187075/package.json#L207)